src/daemon/mod.rs

```diff
@@ -437,89 +410,6 @@ pub(super) async fn start(
         .map(|_| {})
 }
 
-/// If our current chain is below a supported height, we need a snapshot to bring it up
```

Comment:
Why did you move this into `db_util.rs`?

Reply:
It is only used for DB initialization. I can move it back to `mod.rs` or a separate `snapshot_util.rs` to not bloat `mod.rs`, let me know which is preferrable

Reply:
Moved back, see https://github.com/ChainSafe/forest/pull/3365#discussion_r1303987435